core/src/assets/config/config.json

```diff
@@ -1,62 +1,80 @@
 {
   "config": {
-    "features":{
-        "SERVICE_CATALOG":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"servicecatalog.k8s.io"
-              }
-          ]
-        },
-        "SERVICE_CATALOG_ADDONS":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"servicecatalog.kyma-project.io"
-              }
-          ]
-        },
-        "EVENTING":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"eventing.kyma-project.io"
-              }
-          ]
-        },
-        "API_GATEWAY":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"gateway.kyma-project.io"
-              }
-          ]
-        },
-        "APPLICATIONS":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"applicationconnector.kyma-project.io"
-              }
-          ]
-        },
-        "ADDONS":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"addons.kyma-project.io"
-              }
-          ]
-        },
-        "SERVERLESS":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"serverless.kyma-project.io"
-              }
+    "features": {
+      "SERVICE_CATALOG": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "servicecatalog.k8s.io"
+          }
+        ]
+      },
+      "SERVICE_CATALOG_ADDONS": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "servicecatalog.kyma-project.io"
+          }
+        ]
+      },
+      "EVENTING": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "eventing.kyma-project.io"
+          }
+        ]
+      },
+      "API_GATEWAY": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "gateway.kyma-project.io"
+          }
+        ]
+      },
+      "APPLICATIONS": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "applicationconnector.kyma-project.io"
+          }
+        ]
+      },
+      "ADDONS": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "addons.kyma-project.io"
+          }
+        ]
+      },
+      "SERVERLESS": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "serverless.kyma-project.io"
+          }
+        ]
+      },
+      "OBSERVABILITY": {
+        "config": {
+          "links": [
+            [
```

Comment:
why are those arrays? I think it would be more readable to have them as objects with `label` and `path`

Reply:
I agree with @dariadomagala . Can we have and array of objects?